### PR TITLE
www-client/qutebrowser add version bump 1.8.3

### PR DESCRIPTION
**Pull request by GuillaumeSeren**

### CODE REVIEW

**Comment by GuillaumeSeren** [comment]
Here the diff
```diff
RepoMan scours the neighborhood...
  dependency.badindev           10
   dev-python/pygments/pygments-2.2.0.ebuild: DEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7']
   dev-python/pygments/pygments-2.2.0.ebuild: RDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7']
   dev-python/pygments/pygments-2.2.0-r1.ebuild: BDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7']
   dev-python/pygments/pygments-2.2.0-r1.ebuild: RDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7']
   dev-python/pygments/pygments-2.3.1.ebuild: BDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7']
   dev-python/pygments/pygments-2.3.1.ebuild: RDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7']
   dev-python/pygments/pygments-2.4.2.ebuild: BDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7',
      'dev-lang/python:3.8']
   dev-python/pygments/pygments-2.4.2.ebuild: RDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7',
      'dev-lang/python:3.8']
   dev-python/pygments/pygments-2.5.2.ebuild: BDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7',
      'dev-lang/python:3.8']
   dev-python/pygments/pygments-2.5.2.ebuild: RDEPEND: ~amd64-linux(default/linux/amd64/17.0/no-multilib/prefix/kernel-3.2+)
[     '>=dev-lang/python-2.7.5-r2:2.7',
      'dev-lang/python:3.5',
      'dev-lang/python:3.6',
      'dev-lang/python:3.7',
      'dev-lang/python:3.8']
RepoMan sez: "You're only giving me a partial QA payment?
              I'll take it this time, but I'm not happy."
```

File: dev-python/pygments/pygments-2.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	test? (
+		dev-python/nose[${PYTHON_USEDEP}]
+		virtual/ttf-fonts
+	)"
+
+distutils_enable_sphinx doc
+
+python_test() {
+	cp -r -l tests "${BUILD_DIR}"/ || die
+	nosetests --verbosity=3 -w "${BUILD_DIR}"/tests ||
```
Comment (hartwork) [inline dev-python/pygments/pygments-2.5.2.ebuild]: It turns out that upstream has moved from nose to py.test in 2.5.2

Comment (GuillaumeSeren) [reply dev-python/pygments/pygments-2.5.2.ebuild]: I have upgraded the pygment ebuild to use pytest instead of nose @hartwork

**Comment by GuillaumeSeren** [comment]
I have upgraded the branch and delete the indirect deps
```
-       >=dev-python/pygments-2.5.2[${PYTHON_USEDEP}]
```

**Comment by GuillaumeSeren** [comment]
Merged

**Pull request closed without merging**